src/main/java/HelpCommand.java

```diff
@@ -0,0 +1,28 @@
+public class HelpCommand extends Command {
+
+    public static final String COMMAND = "help";
+    public static final String ERROR_MESSAGE = "☹ Sorry, please enter a valid command.\n";
+    public static final String COMMAND_LIST = "\tCommands available:\n\t\t" +
```

Comment:
Perhaps the same breaking into multiple lines argument goes here too to improve readability, and ease of maintenance should the order be changed or new commands be added.